File: Server/schemas/workshop_inscription/workshop_inscription_out.py
from pydantic import BaseModel, Field
from utils.constants import MAX_LENGTH_NAME

class WorkshopInscriptionOut(BaseModel):
    id_inscription: int
    id_workshop: int
    id_student: int
    rut_student: str
    semester: str
    year: int
    is_convalidated: bool
    id_curriculum_course: int
    curriculum_course: str 

File: Client/src/features/business/workshops/types/workshop_inscription_out.ts
// workshop_inscription_out.ts
// Interface de inscripción a taller (output) basada en OUT del backend

export interface WorkshopInscriptionOut {
  id_inscription: number
  id_workshop: number
  id_student: number
  rut_student: string
  semester: string
  year: number
  is_convalidated: boolean
  id_curriculum_course: number
  curriculum_course: string
} 